bitcash/network/APIs/BitcoinDotComAPI.py

```diff
@@ -0,0 +1,132 @@
+import requests
+from decimal import Decimal
+from bitcash.exceptions import InvalidEndpointURLProvided
+from bitcash.network import currency_to_satoshi
+from bitcash.network.meta import Unspent
+from bitcash.network.transaction import Transaction, TxPart
+
+# This class is the interface for Bitcash to interact with
+# Bitcoin.com based RESTful interfaces.
+
+DEFAULT_TIMEOUT = 30
```

Comment:
This should be a much lower value, probably 4 seconds.

Reply:
Fixed in 8f3f746245278788eb09f533a6e4e65172690cab